src/core/public/chrome/ui/header/collapsible_nav_group_enabled.tsx

```diff
@@ -60,14 +60,6 @@ const titleForSeeAll = i18n.translate('core.ui.primaryNav.seeAllLabel', {
   defaultMessage: 'See all...',
 });
 
-// Custom category is used for those features not belong to any of use cases in all use case.
-// and the custom category should always sit after manage category
-const customCategory: AppCategory = {
-  id: 'custom',
-  label: i18n.translate('core.ui.customNavList.label', { defaultMessage: 'Custom' }),
-  order: (DEFAULT_APP_CATEGORIES.manage.order || 0) + 500,
-};
-
 enum NavWidth {
   Expanded = 270,
   Collapsed = 48, // The Collasped width is supposed to be aligned with the hamburger icon on the top left navigation.
```

Comment:
For later: Not a fan of having layout decisions made in JS. We should plan to move these to CSS, as classes that we toggle, or as CSS or SASS variables.